src/common/types/types.cpp

```diff
@@ -1748,5 +1801,68 @@ bool LogicalTypeUtils::tryGetMaxLogicalType(const std::vector<LogicalType>& type
     return true;
 }
 
+LogicalType LogicalTypeUtils::combineTypes(const common::LogicalType& lft,
```

Comment:
If u are introducing this, we need to document under what case we should use getMaxLogicalType and what case we should use combineTypes. How are they different from each other.

Reply:
CombineTypes combines to STRING as the widest type. I'll add this as a comment.